src/main/java/org/jenkinsci/plugins/workflow/cps/GroovyClassLoaderWhitelist.java

```diff
@@ -31,17 +35,17 @@
      */
     private final Whitelist delegate;
 
-    GroovyClassLoaderWhitelist(GroovyClassLoader scriptLoader, Whitelist delegate) {
-        this.scriptLoader = scriptLoader;
+    GroovyClassLoaderWhitelist(Whitelist delegate, GroovyClassLoader... scriptLoaders) {
+        this.scriptLoaders = Arrays.<ClassLoader>asList(scriptLoaders);
```

Comment:
:ant: Make this an `IdentityHashMap<ClassLoader,Void>` so that `permits` is not calling `Object.equals`, which makes me nervous.